test/e2e/run-all.js

```diff
@@ -67,6 +67,14 @@ async function runningOnCircleCI(testPaths) {
   const changedOrNewTests = await filterE2eChangedFiles();
   console.log('Changed or new test list:', changedOrNewTests);
 
+  const hasOnlyMdOrCsvChanges = await checkOnlyMdOrCsvFiles();
+  if (hasOnlyMdOrCsvChanges) {
```

Comment:
I got LLM help over here and not sure if this will work, feedbacks are appreciated 🙏